### ISSUE TITLE
PIXI.js v5 compatibility

### ISSUE THREAD

**Issue by kulabros** [ymiroshnyk/neutrinoparticles.js#6]
Hi there, I'm trying to run particles on new v5 pre release versions of pixi.js and it seems it doesn't work, PIXINeutrinoEffect is instanced, but it's effect property if null.. Is there gonna be any engine update for v5? Thanks for info

**Comment by ymiroshnyk** [comment]
Hi. I didn't make any efforts related to pixi v5 since it is not released yet.
Now when I have first request about it, I'll put it in the schedule :)
Thanks

**Comment by kulabros** [comment]
Of course, I was just trying for the first time new pixi.js v5 alpha version, what's new and hit this issue. Very cool, good luck! :)

**Comment by knifhen** [comment]
Hi, do you have plans for making the neutrinoparticles.js work with the now released pixi v5?

Best regards Andreas Knifh

**Comment by ymiroshnyk** [comment]
Oh, my God! It is released! :)
Yes, I'll make an effort on creating V5 renderer shortly.

Thanks for pointing on that.

**Comment by arrudenk** [comment]
@ymiroshnyk so, how it's going ?:)

**Comment by ymiroshnyk** [comment]
@arrudenk Work in progress. See branch pixi5-dev. For now integration looks like completed. Further I need to check everything though. Also interface may change slightly till the end of integration.
I plan to finish it till the end of the week, so you will be able to beta-test it :)

**Comment by ymiroshnyk** [comment]
@arrudenk you can check current version. It has WebGL support only. There are several samples to show basic usage. And I'll investigate Canvas support in pixi5.

**Comment by ymiroshnyk** [comment]
Done with CanvasRenderer. Samples modified to have a flag to force it enabled. Interface is slightly changed. Compiled library committed to dist-PIXI/neutrinoparticles.pixi5.js (+map).

I believe everything is ready to try in production. All my tests are passed well.

**Comment by arrudenk** [comment]
@ymiroshnyk nice to hear I will try it
